.changes/unreleased/Added-20240917-214812.yaml

```
@@ -0,0 +1,6 @@
+kind: Added
+body: Added "container.up" api
```

```suggestion
body: Added `Container.up` api
```

I'd also recommend adding a second line that explains that it's an alias to `.AsService().Up()`.